socorro/external/rabbitmq/crashstorage.py

```diff
@@ -210,14 +211,20 @@ def _consume_acknowledgement_queue(self):
             pass  # nothing to do with an empty queue
 
     #--------------------------------------------------------------------------
-    def _transaction_ack_crash(self, connection, acknowledgement_token):
-        self.config.logger.debug(
-            'RabbitMQCrashStorage acking with delivery_tag %s',
-            acknowledgement_token.delivery_tag
-        )
+    def _transaction_ack_crash(
+        self,
+        connection,
+        crash_id,
+        acknowledgement_token
+    ):
         connection.channel.basic_ack(
             delivery_tag=acknowledgement_token.delivery_tag
         )
+        self.config.logger.debug(
```

Comment:
Should be put a `logger.debug(...` **before** the `connection.channel.basic_ack(...)` so that you can see crashes that are **attempted** to be acked. Then you might be able to see which ones are attempted but don't get the `'RabbitMQCrashStorage acking %s with delivery_tag %s'` line.